gitlab-modules/multiple-pipline-same-repo/README.md

```diff
@@ -0,0 +1,16 @@
+### Setup
+
+#### Project Info
+Replace below configuration with your project details:
+
+Project group: baeldung-developers
+Project name: demo
+Project ID: 61297443
```

Comment:
I'm not clear what you mean here, I can't see this configuration used anywhere?

Reply:
You're right; these values aren't referred to in the configuration anywhere.
To avoid confusion, we've removed these values from the `README.md` file.